solcx/install.py

```diff
@@ -370,10 +381,8 @@ def _install_solc_osx(version, allow_osx, show_progress):
 
 
 if __name__ == "__main__":
-    try:
-        version = sys.argv[1]
-    except IndexError:
-        LOGGER.error("Invocation error.  Should be invoked as `./install_solc.py <release-tag>`")
-        sys.exit(1)
-
-    install_solc(version)
+    argument_parser = argparse.ArgumentParser()
```

Comment:
Thank you for swapping this to `argparse` as well, it was long overdue.

Reply:
Welcome :)